external/vcm/tests/test_coarsen_restarts.py

```diff
@@ -112,7 +113,11 @@ def test_coarsen_restarts(tag):
     restart_data, grid_data = generate_synthetic_data(restart_schemas, grid_spec_schema)
 
     func, kwargs = REGRESSION_TESTS[tag]
-    result = func(FACTOR, grid_data, restart_data, **kwargs)
+    if func == coarsen_restarts_on_sigma:
+        # coarsen_restarts_on_sigma does not require the toa_pressure as an argument
+        result = func(FACTOR, grid_data, restart_data, **kwargs)
+    else:
+        result = func(FACTOR, grid_data, TOA_PRESSURE, restart_data, **kwargs)
```

Comment:
[optional] the other option is to have `coarsen_restarts_on_sigma` still take in `toa_pressure` and do nothing. I find this if statement to be a bit awkward, or perhaps this test should be split into different tests.